android/src/main/java/com/hieuvp/fingerprint/ReactNativeFingerprintScannerModule.java

```diff
@@ -110,7 +110,7 @@ public void isSensorAvailable(final Promise promise) {
         if (errorMessage != null) {
             promise.reject(errorMessage, errorMessage);
         } else {
-            promise.resolve(true);
+            promise.resolve('Fingerprint');
```

Comment:
This will break android build. Java doesn't treat it like a string.  Instead it treat it as string literal. Use double quotes to fix.